feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64fafac95328110a69bcb75f.md

---
id: 64fafac95328110a69bcb75f
title: Step 38
challengeType: 0
dashedName: step-38
---

# --description--

> Use arrow syntax to create a `editTask` function. Pass in `buttonEl` as the parameter and have an empty curly braces.

Reviewer: ```suggestion
Use arrow syntax to create an `editTask` function. Pass in `buttonEl` as the parameter and add empty curly braces for the body.
```
